src/RevOpts.cc

```diff
@@ -12,29 +12,6 @@
 
 namespace SST::RevCPU {
 
-RevOpts::RevOpts( uint32_t NumCores, uint32_t NumHarts, const int Verbosity )
-  : numCores( NumCores ), numHarts( NumHarts ), verbosity( Verbosity ) {
-
-  std::pair<uint32_t, uint32_t> InitialPair;
-  InitialPair.first  = 0;
-  InitialPair.second = 10;
-
-  // init all the standard options
-  // -- startAddr = 0x00000000
-  // -- machine = "G" aka, "IMAFD"
-  // -- pipeLine = 5
-  // -- table = internal
-  // -- memCosts[core] = 0:10
-  // -- prefetch depth = 16
-  for( uint32_t i = 0; i < numCores; i++ ) {
-    startAddr.insert( std::pair<uint32_t, uint64_t>( i, 0 ) );
-    machine.insert( std::pair<uint32_t, std::string>( i, "G" ) );
-    table.insert( std::pair<uint32_t, std::string>( i, "_REV_INTERNAL_" ) );
-    memCosts.push_back( InitialPair );
-    prefetchDepth.insert( std::pair<uint32_t, uint32_t>( i, 16 ) );
-  }
-}
-
```

Comment:
All of that loop has been moved to the constructor initialization syntax above, where `std::vector(numProcs, value)` initializes the vector to length `numProcs` copies of `value`.